src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em;
```

Comment:
you don't need left and right margins

Reply:
done but i have some bug , because in my version i have margin: 1em 0;